client/main.lua

```diff
@@ -144,6 +114,84 @@ local function DeliverAnim()
         garbageObject = nil
         canTakeBag = true
     end)
+    if Config.UseTarget and hasBag then
+        local CL = Config.Locations["trashcan"][currentStop]
+        hasBag = false
+        local pos = GetEntityCoords(ped)
+        exports['qb-target']:RemoveTargetEntity(garbageVehicle)
+        if (amountOfBags - 1) <= 0 then
+            QBCore.Functions.TriggerCallback('garbagejob:server:NextStop', function(hasMoreStops, nextStop, newBagAmount)
+                if hasMoreStops and nextStop ~= 0 then
+                    -- Here he puts your next location and you are not finished working yet.
+                    currentStop = nextStop
+                    currentStopNum = currentStopNum + 1
+                    amountOfBags = newBagAmount
+                    SetGarbageRoute()
+                    QBCore.Functions.Notify(Lang:t("info.all_bags"))
+                    SetVehicleDoorShut(garbageVehicle, 5, false)
+                else
+                    if hasMoreStops and nextStop == currentStop then
+                        QBCore.Functions.Notify(Lang:t("info.depot_issue"))
+                        amountOfBags = 0
+                    else
+                        -- You are done with work here.
+                        QBCore.Functions.Notify(Lang:t("info.done_working"))
+                        SetVehicleDoorShut(garbageVehicle, 5, false)
+                        RemoveBlip(deliveryBlip)
+                        SetRouteBack()
+                        amountOfBags = 0
+                    end
+                end
+            end, currentStop, currentStopNum, pos)
+        else
+            -- You haven't delivered all bags here
+            amountOfBags = amountOfBags - 1
+            if amountOfBags > 1 then
+                QBCore.Functions.Notify(Lang:t("info.bags_left", { value = amountOfBags }))
+            else
+                QBCore.Functions.Notify(Lang:t("info.bags_still", { value = amountOfBags }))
+            end
+            exports['qb-target']:AddCircleZone('garbagebin', vector3(CL.coords.x, CL.coords.y, CL.coords.z), 2.0,{
+                name = 'garbagebin', debugPoly = false, useZ=true}, {
+                options = {{label = Lang:t("target.grab_garbage"),icon = 'fa-solid fa-trash', action = function() TakeAnim() end}},
+                distance = 2.0
+            })
+        end
+    end
+end
+
+function TakeAnim()
```

Comment:
Indentation here is not yet fixed. The rest of the file uses 4 spaces for indentation, your function uses tabs. Please convert the tabs to 4 spaces.